content/200-concepts/100-components/02-prisma-client/062-computed-fields.mdx

```diff
@@ -10,6 +10,81 @@ Computed fields allow you to derive a new field based on existing data. A common
 
 </TopBlock>
 
+## Computed fields with Prisma Client extensions
+
+> Prisma Client extensions are currently in [Preview](/about/prisma/releases#preview).
+
+The following example illustrates how to create a [Prisma Client extension](/concepts/components/prisma-client/client-extensions) that adds a virtual/ computed field to a model in a Prisma schema. The example adds a `fullName` property that is computed at runtime.
+
+<TabbedContent tabs={[<FileWithIcon text="Prisma Client extension" icon="code"/>, <FileWithIcon text="Example query" icon="code"/>,<FileWithIcon text="Prisma schema" icon="file"/>]}>
+
+<tab>
+
+```tsx
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient().$extends({
+  result: {
+    user: {
+      fullName: {
+        needs: { firstName: true, lastName: true },
+        compute(user) {
+          return `${user.firstName} ${user.lastName}`
+        },
+      },
+    },
+  },
+})
+```
+
+</tab>
+
+<tab>
+
+```ts
+const user = await prisma.user.findMany({
+  take: 5,
+})
+```
+
+</tab>
+
+<tab>
+
+```prisma file="prisma/schema.prisma" copy
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["clientExtensions"]
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+
+  posts Post[]
+}
+
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  published Boolean @default(true)
+  content   String?
+
+  authorId Int?
+  author   User? @relation(fields: [authorId], references: [id])
+}
+```
+
+</tab>
+
+</TabbedContent>
+
 ## Deriving a Full Name from a First and Last Name
```

Comment:
I know that this isn't the part that you've modified but when working on a docs page, let's try to look at it holistically and see what we can improve. since we use regular sentence casing for headings, these are small wins to make the docs more consistent.